mcs/class/corlib/ReferenceSources/RuntimeType.cs

```diff
@@ -812,5 +812,20 @@ public override int GetHashCode()
 				return false;
 			}
 		}
+
+		[System.Runtime.InteropServices.ComVisible(true)]
+		[Pure]
+		public override bool IsSubclassOf(Type type)
+		{
+			if ((object)type == null)
+				throw new ArgumentNullException("type");
+			Contract.EndContractBlock();
+			RuntimeType rtType = type as RuntimeType;
+			if (rtType == null)
+				return false;
+
+			return RuntimeTypeHandle.IsSubclassOf (this, rtType);
+		}
```

Comment:
90% of this method is the same as the referencesource version so I'd keep this there and just put the `RuntimeTypeHandle.IsSubclassOf ` in the ifdef.

(probably not worth going through another CI iteration cycle for this, unless you need to make changes anyway)